1 test fails: locale.Error: unsupported locale setting

**Describe the bug**
```
========================================================================================= FAILURES ==========================================================================================
________________________________________________________________________________________ test_locale ________________________________________________________________________________________

    def test_locale():
        import locale
    
        m = Model()
        x = m.addVar(lb=1.1)
    
>       locale.setlocale(locale.LC_NUMERIC, "pt_PT")

tests/test_model.py:352: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

category = 4, locale = 'pt_PT'

    def setlocale(category, locale=None):
    
        """ Set the locale for the given category.  The locale can be
            a string, an iterable of two strings (language code and encoding),
            or None.
    
            Iterables are converted to strings using the locale aliasing
            engine.  Locale strings are passed directly to the C lib.
    
            category may be given as one of the LC_* values.
    
        """
        if locale and not isinstance(locale, _builtin_str):
            # convert to string
            locale = normalize(_build_localename(locale))
>       return _setlocale(category, locale)
E       locale.Error: unsupported locale setting

/usr/local/lib/python3.9/locale.py:610: Error
===================================================================================== warnings summary ======================================================================================
tests/test_model.py::test_solve_concurrent
  /usr/ports/math/py-PySCIPOpt/work-py39/PySCIPOpt-5.0.1/tests/test_model.py:89: UserWarning: SCIP was compiled without task processing interface. Parallel solve not possible - using optimize() instead of solveConcurrent()
    s.solveConcurrent()

tests/test_pricer.py::test_incomplete_pricer
  /usr/local/lib/python3.9/site-packages/_pytest/unraisableexception.py:78: PytestUnraisableExceptionWarning: Exception ignored in: 'pyscipopt.scip.PyPricerRedcost'
  
  Traceback (most recent call last):
    File "src/pyscipopt/pricer.pxi", line 28, in pyscipopt.scip.Pricer.pricerredcost
      raise NotImplementedError("pricerredcost() is a fundamental callback and should be implemented in the derived class")
  NotImplementedError: pricerredcost() is a fundamental callback and should be implemented in the derived class
  
    warnings.warn(pytest.PytestUnraisableExceptionWarning(msg))

tests/test_relax.py::test_empty_relaxator
  /usr/local/lib/python3.9/site-packages/_pytest/unraisableexception.py:78: PytestUnraisableExceptionWarning: Exception ignored in: 'pyscipopt.scip.PyRelaxExec'
  
  Traceback (most recent call last):
    File "/usr/ports/math/py-PySCIPOpt/work-py39/PySCIPOpt-5.0.1/tests/test_relax.py", line 64, in test_empty_relaxator
      m.optimize()
  AssertionError: relaxexec() must return a dictionary.
  
    warnings.warn(pytest.PytestUnraisableExceptionWarning(msg))

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================================================================== short test summary info ==================================================================================
SKIPPED [1] tests/test_branch_incomplete.py:5: fix later
SKIPPED [1] tests/test_cons.py:155: TODO: test getValsLinear()
SKIPPED [1] tests/test_cons.py:160: TODO: test getRowLinear()
SKIPPED [1] tests/test_heur.py:53: Skipped
SKIPPED [96] tests/test_logical.py:162: unsupported logical: B
SKIPPED [1] tests/test_memory.py:7: Skipped
SKIPPED [1] tests/test_memory.py:13: Skipped
SKIPPED [1] tests/test_nonlinear.py:111: Test fails on CPython3.6 for MacOS with x86_64
SKIPPED [1] tests/test_short.py:62: got empty parameter set ['instance'], function test_instance at /usr/ports/math/py-PySCIPOpt/work-py39/PySCIPOpt-5.0.1/tests/test_short.py:61
======================================================= 1 failed, 274 passed, 104 skipped, 11 xfailed, 3 warnings in 80.01s (0:01:20) =======================================================
```

**To Reproduce**
Regular build+tests.

**System**
 - OS: FreeBSD 14.0
 - Version: 5.0.1
 - SCIP version: 9.0.0

--- a/.github/workflows/integration-test.yml
+++ b/.github/workflows/integration-test.yml
@@ -25,9 +25,6 @@ jobs:
           wget --quiet --no-check-certificate https://github.com/scipopt/scip/releases/download/$(echo "v${{env.version}}" | tr -d '.')/SCIPOptSuite-${{ env.version }}-Linux-ubuntu20.deb
           sudo apt-get update && sudo apt install -y ./SCIPOptSuite-${{ env.version }}-Linux-ubuntu20.deb
 
-      - name: Install locales for tests
-        run: sudo apt-get install tzdata locales -y && sudo locale-gen pt_PT && sudo update-locale # add pt_PT locale that is used in tests
-
       - name: Setup python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,7 @@
 
 ## Unreleased
 ### Added
+- Added check for pt_PT locale in test_model.py
 ### Fixed
 ### Changed
 ### Removed

--- a/tests/test_model.py
+++ b/tests/test_model.py
@@ -347,12 +347,17 @@ def test_locale():
     import locale
 
     m = Model()
-    x = m.addVar(lb=1.1)
-
-    locale.setlocale(locale.LC_NUMERIC, "pt_PT")
+    m.addVar(lb=1.1)
+    
+    try:
+        locale.setlocale(locale.LC_NUMERIC, "pt_PT")
+    except Exception:
+        pytest.skip("pt_PT locale is non-default and needs to be installed.")
+    
 
     assert locale.str(1.1) == "1,1"
 
+    
     m.writeProblem("model.cip")
 
     with open("model.cip") as file:

--- a/tests/test_model.py
+++ b/tests/test_model.py
@@ -352,11 +352,9 @@ def test_locale():
     try:
         locale.setlocale(locale.LC_NUMERIC, "pt_PT")
     except Exception:
-        pytest.skip("pt_PT locale is non-default and needs to be installed.")
+        pytest.skip("pt_PT locale was not found. It might need to be installed.")
     
-
     assert locale.str(1.1) == "1,1"
-
     
     m.writeProblem("model.cip")
 

